app/views/layouts/_analytics.html.erb

```diff
@@ -0,0 +1,13 @@
+<!-- We participate in the US government's analytics program. See the data at analytics.usa.gov. -->
+<script id="_fed_an_ua_tag" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js"></script>
+<!-- 18F analytics -->
+<script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+  ga('create', 'UA-48605964-27', 'auto');
```

Comment:
Probably a good idea to set this as an environmental variable/setting, if only to allow us swap it out in different environments.